torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6505,6 +6514,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
 
 # Common operator groupings
 unary_ufuncs = [op for op in op_db if isinstance(op, UnaryUfuncInfo)]
+test_funcs = [op for op in op_db if isinstance(op, OpInfo)]
```

Comment:
This won't filter out anything as all entries are derived from `OpInfo`. Just use `op_db` in `test_ops.py`

Reply:
True! Was checking what's the difference, I'll remove these unnecessary lines (also in the TODO). :) Thanks!

Reply:
I'm surprised I didn't remove this yet, apologies. I'll do remove this in the next commit.

Reply:
Just be sure to keep conversations unresolved until the change appears to make it easier to follow the PR